### PR TITLE
[WIP] implement return game list for switch game.

### PR DESCRIPTION
**Pull request by Zangetsu38**
WIP warning
- Implement possible exit app during it running for return on game list with returnb on live area game screen

# Result:
- App rurned 
image

### CODE REVIEW

File: vita3k/gui/include/gui/functions.h [hunk 1]
```diff
@@ -49,7 +49,8 @@ void load_game_background(GuiState &gui, HostState &host);
 bool refresh_game_list(GuiState &gui, HostState &host);
 
 void draw_begin(GuiState &gui, HostState &host);
-void draw_end(GuiState &host, SDL_Window *window);
+void draw_end(GuiState &gui, SDL_Window *window);
```
Comment (Zangetsu38) [inline vita3k/gui/include/gui/functions.h]: i have see this fail, no idea what is set wrong

File: vita3k/main.cpp [hunk 2]
```diff
@@ -117,66 +117,77 @@ int main(int argc, char *argv[]) {
...
 
-    if (!gl_renderer.init(host.base_path))
-        return RendererInitFailed;
+            gl_renderer.render(host);
 
-    while (handle_events(host, gui)) {
-        // Driver acto!
-        renderer::process_batches(*host.renderer.get(), host.renderer->features, host.mem, host.cfg, host.base_path.c_str(),
-            host.io.title_id.c_str());
+            // Calculate FPS
```
Comment (Margen67) [inline vita3k/main.cpp]: This comment is unnecessary.

Comment (Zangetsu38) [reply vita3k/main.cpp]: this comment ,is not me, i have unctouched it, is master code

Comment (pent0) [reply vita3k/main.cpp]: bruh leave it there

**Comment by Zangetsu38** [comment]
Obsolete pr closing it

**Pull request closed without merging**